addons/l10n_in_edi/models/res_company.py

```diff
@@ -16,9 +16,14 @@ class ResCompany(models.Model):
         help="Enable the use of production credentials",
         groups="base.group_system",
     )
+    l10n_in_edi_token_status = fields.Boolean(compute="_compute_l10n_edi_token_status")
```

Comment:
Why does the user have to intervene for this?  Is not the update of the token happening in the background by the system anyways?

Reply:
when the user opens the setting then the user doesn't know if it's verified or not.
most of the time happens when more than one admin in the system
so every time need to click verify button to check if it's valid or not.